examples/stream-in/data.csv

```diff
@@ -0,0 +1,5244 @@
+a,b,c,d,e,f,g
```

Comment:
Add this to the .npmignore?

Reply:
I'll add a script to generate it and put it in .gitignore and .npmignore